### DISCUSSION TITLE
Cucumber support

### DISCUSSION THREAD

**Discussion by psa83** [mmisty/cypress-allure-adapter#83] [General]
Hello,
Any prevision when the cucumber support will be available?

Regards,
Pedro Sá

**Comment by mmisty** [comment]
Hello, @fuze-psa 

 it has been implemented in [0.14.1](https://github.com/mmisty/cypress-allure-adapter/blob/main/CHANGELOG.md#0141), current version is 0.15.2.
You can check usage example [here](https://github.com/mmisty/cypress-allure-adapter-example/tree/main/examples/cypress-js-gherkin)

**Reply by psa83** [reply]
@mmisty Thank you.
